google/generativeai/files.py

```diff
@@ -52,14 +55,20 @@ def upload_file(
     return file_types.File(response)
 
 
-def list_files(page_size=50) -> Iterable[file_types.File]:
+def list_files(max_results=50) -> Iterable[file_types.File]:
     client = get_default_file_client()
 
-    response = client.list_files(page_size=page_size)
+    response = client.list_files(glm.ListFilesRequest())
```

Comment:
Oops, i had assumed you had used a `option (google.api.method_signature) = "page_size";` here. The page size can still go in this way:

```suggestion
    response = client.list_files(glm.ListFilesRequest(page_size=page_size))
```